projects/dev-schematics/scripts/outlet-template-files.sh

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
Can we document how to use this script in the `README.md`?
A few words on what it does, and a few examples would be nice.